deployment/terraform/lambda.tf

```diff
@@ -0,0 +1,56 @@
+resource "aws_lambda_event_source_mapping" "kf_lambda_source_mapping" {
+  event_source_arn = aws_sqs_queue.kf_lambda_sqs.arn
+  function_name    = aws_lambda_function.kf_lambda_function.function_name
+}
+
+data "aws_ecr_repository" "ecr" {
+  name = var.ecr_name
+}
+
+resource "aws_lambda_function" "kf_lambda_function" {
+  #checkov:skip=CKV_AWS_272:No definition for this rule
+  #checkov:skip=CKV_AWS_116:TODO-Add dead-letter queue
+  #ts:skip=AWS.LambdaFunction.LM.MEIDUM.0063 Policy is defined in attached lambda role
```

Comment:
I'm skipping a few rules here for checkov and terrascan, but I think the only one we will need to address at some point is [CKV_AWS_116](https://docs.bridgecrew.io/docs/ensure-that-aws-lambda-function-is-configured-for-a-dead-letter-queue-dlq) and that may not be a priority.